### PR TITLE
ENH: ReorientImage interface

### PR DESCRIPTION
**Pull request by mgxd**
Ports over a new interface implemented in https://github.com/nipreps/nibabies/pull/229

No rush in getting this in, would like a review whenever vacations are over

### CODE REVIEW

File: niworkflows/interfaces/tests/test_nibabel.py [hunk 1]
```diff
@@ -280,3 +282,55 @@ def test_map_labels(tmpdir, data, mapping, tojson, expected):
...
+    target_img = nb.load(target_file)
+    assert ''.join(nb.aff2axcodes(target_img.affine)) == out_ornt
+    res = ReorientImage(in_file=in_file, target_file=target_file).run()
+    out_file = res.outputs.out_file
+    out_img = nb.load(out_file)
+    assert ''.join(nb.aff2axcodes(out_img.affine)) == out_ornt
+
+    # cleanup
+    for f in (in_file, target_file, out_file):
+        Path(f).unlink()
```
Comment (oesteban) [inline niworkflows/interfaces/tests/test_nibabel.py:336]: is this necessary when using the tmpdir fixture?

**Review by oesteban — APPROVED** [review]
LGTM - more readable, tested, thanks!

File: niworkflows/interfaces/tests/test_nibabel.py (hunk 1 above)
Comment (mgxd) [reply niworkflows/interfaces/tests/test_nibabel.py:336]: probably not 😅 - just figured I'd clean it anyways. I can remove it if you feel strongly

**Comment by mgxd** [comment]
I realized the CIFTI function was relying on the reorientation being in memory, so I separated the reorientation into file based and memory based functions.

**Pull request merged**